api/supported_series.yaml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/reubenmiller/go-c8y-cli/feat/extensions-manager/tools/schema/extensionCommands.json
+---
+group:
+  name: supportedseries
+  description: OEE supported series
+
+commands:
+  - name: list
+    description: Get all supported series for the device
+    descriptionLong: This service will take the deviceId as input and return the all supported series descriptors.
+    method: GET
+    path: service/oee-bundle/supportedseries/{device}
+    exampleList:
+      - command: c8y oee supportedSeries --device 12345 --dateFrom 1996-03-01T22:52:50.493Z --dateTo 1996-03-01T23:52:50.493Z
+        description: Get support series for device with id 12345 within period from 1996-03-01T22:52:50.493Z to 1996-03-01T23:52:50.493Z
+    pathParameters:
+      - name: device
+        type: device[]
+        description: Device
+        pipeline: true
+        required: true
+    queryParameters:
```

Comment:
- both date parameters are marked as required, but the CLI doesnt show that; I can execute the command and get a 400 error back; I'd expect a similar message as shown when the device is not provided
```
PS C:\Users\mvoigt> c8y oee-cli.naph supportedseries list
2023-05-31T09:12:31.513+0200    ERROR   commandError: missing required parameters. [device]
PS C:\Users\mvoigt> c8y oee-cli.naph supportedseries list --device Ideal` Producer` A80
2023-05-31T09:12:40.897+0200    ERROR   serverError: GET https://mobile-mvoigt.eu-latest.cumulocity.com/service/oee-bundle/supportedseries/100104815: 400
```